.github/workflows/default.yml

```diff
@@ -115,9 +120,6 @@ jobs:
           - gradle: 7.0.1
             java: 11
             agp: 4.2.2
-          - gradle: 6.9.1
-            java: 8
-            agp: 4.0.0
```

Comment:
AGP introduced `namespace` property, which is available since 4.2 and is required starting from 8.0.0. Given the above, this part of the matrix failed, as the `namespace` I'm adding below isn't supported 👀 
I didn't want to make our configuration too complicated, so I'm proposing to stop testing on that old version 🤷 It doesn't change plugin compatibility, but given we'll stop testing on it - it's not guaranteed it _will_ work.